db/models.go

```diff
@@ -0,0 +1,57 @@
+package db
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Player struct {
+	gorm.Model
+	Name string `gorm:"unique;not null"`
+
+	PlayScores []PlayScore
+}
+
+type Play struct {
+	gorm.Model
+	Place string
+	Date  time.Time
+
+	PlayScores []PlayScore
+}
+
+// TODO{mikitachab} add wonders type enum
+type SevenWondersScore struct {
+	gorm.Model
+	WonderName           string
+	MilitaryConflicts    int
+	Wonders              int
+	TreasuryContents     int
+	CivilianStructures   int
+	ScientificStructures int
+	CommercialStructures int
+	Guilds               int
+
+	PlayScores []PlayScore
+}
+
+func (sws *SevenWondersScore) ScoreSum() int {
```

Comment:
GetScoreSum or CalculateScoreSum, since "score" is a verb, might be a little confusing later on.

Reply:
yeah, I should change it

Reply:
done